src-backend/api/views.py

```diff
@@ -37,6 +39,33 @@ def get_queryset(self):
         user = self.request.user
         return models.Page.objects.filter(procedure__owner_id__exact=user.id)
 
+    def get_serializer(self, *args, **kwargs):
```

Comment:
Removed when I realized that I no longer needed it since it will always be an array being return to me